docs/05-technical-reference/03-architecture/evnt-02-event-processing.md

```diff
@@ -5,7 +5,7 @@ type: Architecture
 The event processing and delivery flow uses the NATS server to process events and send them to subscribers.
 This diagram explains the event flow in Kyma, from the moment the Application sends an event, to the point when the event triggers the Function.
 
-![Eventing flow](./assets/eventing-flow.svg)
+![Eventing flow](./assets/evnt-processing-flow.svg)
 
 1. The Eventing Controller watches the Subscription Custom Resource. It determines if there are any new incoming events.
```

Comment:
In point no.3 you've got a link that will not work. Remember to change it after @majakurcius adds the Application Connector docs.